src/Table/index.jsx

```diff
@@ -73,14 +170,22 @@ Table.propTypes = {
       PropTypes.string,
       PropTypes.element,
     ]).isRequired,
+    columnSortable: isRequiredIf(PropTypes.bool, props => props.tableSortable),
```

Comment:
I don't believe that this conditional requirement works correctly, since it refers outside of column. I've been able to exclude this prop when initializing a sortable table without propTypes complaining.

Reply:
I think so. I'm not positive, because the [documentation](https://github.com/evcohen/react-proptype-conditional-require) isn't super clear. The conditional (props => props.tableSortable in this case) has access to props, propName, and componentName. If I console.log those from the conditional function, props is

![image](https://user-images.githubusercontent.com/11871801/32246724-4bc55ece-be56-11e7-8ed4-f42b274ac189.png)

which are the props for the column and not the actual Table :(. Should I remove the isRequiredIf if it's not really doing anything? Should I replace it with something? A note in the docs?

Reply:
Just to follow up: this looks like an issue with the actual library and I'm going to see if I can help patch it upstream.